app/src/main/java/com/rage/clamber/Adapters/ClimbsRecyclerViewAdapter.java

```diff
@@ -0,0 +1,85 @@
+package com.rage.clamber.Adapters;
+
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.CheckBox;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.rage.clamber.R;
+
+import butterknife.Bind;
+import butterknife.ButterKnife;
+
+/**
+ * Adapter to display the rows of climbs.
+ */
+public class ClimbsRecyclerViewAdapter extends RecyclerView.Adapter<ClimbsRecyclerViewAdapter.ClimbsViewHolder> {
+
+    //TODO: Set up a listener interface
+
+    String [] climbGrades;
+
+    public ClimbsRecyclerViewAdapter(String [] climbs) {
+        climbGrades = climbs;
+    }
+
+
+    @Override
+    public ClimbsViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+        LayoutInflater inflater = LayoutInflater.from(parent.getContext());
+        View climbsView = inflater.inflate(R.layout.climb_row, parent, false);
+        return new ClimbsViewHolder(climbsView);
+    }
+
+    @Override
+    public void onBindViewHolder(ClimbsViewHolder holder, int position) {
+        String oneClimb = climbGrades[position];
+        holder.gradeDataTextView.setText(oneClimb);
+    }
+
+    @Override
+    public int getItemCount() {
+        return climbGrades.length;
+    }
+
+    static class ClimbsViewHolder extends RecyclerView.ViewHolder {
```

Comment:
final reversal: apparently the standard is to make these classes `public static class` so let's go with that for all of the `ViewHolder`s